src/shared/bus-unit-util.h

```diff
@@ -7,6 +7,14 @@
 #include "pidref.h"
 #include "unit-def.h"
 
+typedef enum ExecDirectoryFlags {
+        EXEC_DIRECTORY_READ_ONLY   = 1 << 0, /* Public API via DBUS, do not change */
+        EXEC_DIRECTORY_ONLY_CREATE = 1 << 1,
```

Comment:
hmm, can you extend the comment on this flag?

Reply:
Extend in which way?

Reply:
I tried to understand what the 2nd flag does, and i didn't get it really. would love a comment. i know this is a pre-existing problem, but if you touch this, please add this.

the concept was added in a2ab603cc42e1484c799f76a233b077c17db91cb.

Reply:
I just changed from a bool to a flag, I didn't add this, if you tell me what to write I'll add it, but I'm not really sure either

Reply:
Sigh. I linked you the commit msg, just copy the relevant last paragraph from there.